portfolio/pom.xml

```diff
@@ -33,6 +33,11 @@
       <artifactId>gson</artifactId>
       <version>2.8.6</version>
     </dependency>
+    <dependency>
```

Comment:
Do we still need this? :( Sorry I know we had issues getting it working. 

I heard it might work when deployed to app engine when using the client libraries https://cloud.google.com/appengine/docs/flexible/python/access-control#apps. But haven't tried myself. I did try the CURL request on the quickstart https://cloud.google.com/translate/docs/basic/setup-basic#make_a_translation_api_request so I know that won't work for sure.

Reply:
You are right, I will remove the dependency for now. That sounds like it may be promising in the future though!